rfc/rfc003-oauth2-authorization.md

```diff
@@ -241,7 +198,7 @@ The **aud** field MUST match the registered endpoint from which the authorizatio
 
 **5.2.1.7. Validate legal base**
 
-The **iss** fields contains the identifier of the actor, the **sub** field contains the identifier of the custodian and the **sid** field contains the identifier for the subject. A known legal base MAY be present at the authorization server/resource server side for this triple. The requested scopes MAY be present in a legal base. Any scope requests that do not follow a legal base MUST NOT be put in the response. If a particular resource may be accessed MUST be checked when accessing the resource. If no **sid** field is present, this check MUST be skipped.
+The **iss** fields contains the identifier of the actor, the **sub** field contains the identifier of the custodian and the **sid** field contains the identifier for the subject. A known legal base MAY be present at the authorization server/resource server side for this triple. If a particular resource may be accessed MUST be checked when accessing the resource. If no **sid** field is present, this check MUST be skipped. An error MUST be returned when this validation fails.
```

Comment:
shoudln't we specified which error MUST be returned?

Reply:
shall add invalid_request